docs/articles/mongodb-administration.md

```diff
@@ -1,4 +1,4 @@
-# Using an external MongoDB instance
+# MongoDB administration
```

Comment:
suggest to put the old content under a "Using an external MongoDB instance" headline